gserver/test/data/steps/pureTextSteps.steps.js

```diff
@@ -1 +1,2 @@
 this.When('I give 3/4 and 5$')
+this.When('I ask {string} and {string} to {word}')
```

Comment:
nope it concerns both values of pureTextSteps

Reply:
If this action is not related to the `pureTextSteps`, then it would be better to add this step to the `gserver/test/data/steps/cucumberExpressions.steps.js` and add the corresponding test to the or `gserver/test/cucumberExpressions.steps.handler.spec.ts` or just add test in the `describe('getCompletion'` test. You could do this or I could make it later in this PR scope.